src/google/protobuf/compiler/java/java_message.cc

```diff
@@ -124,7 +124,7 @@ void ImmutableMessageGenerator::GenerateStaticVariables(io::Printer* printer) {
 
     // The descriptor for this type.
     printer->Print(vars,
-      "$private$static final com.google.protobuf.Descriptors.Descriptor\n"
+      "$private$static com.google.protobuf.Descriptors.Descriptor\n"
```

Comment:
The problem with that is that I *am* Daniel Martin. I have two email addresses - with different Google accounts - and I've signed the CLA with both of them, pointing to the same github account. Apparently the CLA bot can't handle that, so I'll redo all the commits with the fizbin@gmail address.